include/exiv2/slice.hpp

```diff
@@ -429,24 +429,11 @@ struct Slice : public Internal::MutableSliceBase<Internal::ContainerStorage, con
   using value_type = typename std::remove_cv<typename container::value_type>::type;
 #endif
 
-  /*!
-   * Construct a sub-slice of this slice with the given bounds. The bounds
-   * are evaluated with respect to the current slice.
-   *
-   * @param[in] begin  First element in the new slice.
-   * @param[in] end  First element beyond the new slice.
-   *
-   * @throw std::out_of_range when begin or end are invalid
-   */
-  Slice subSlice(size_t begin, size_t end) {
-    return Internal::MutableSliceBase<Internal::ContainerStorage, container>::template subSlice<Slice>(begin, end);
-  }
-
   /*!
    * Constructs a new constant subSlice. Behaves otherwise exactly like
    * the non-const version.
    */
-  [[nodiscard]] Slice<const container> subSlice(size_t begin, size_t end) const {
+  Slice<const container> subSlice(size_t begin, size_t end) const {
```

Comment:
Why did you remove `[[nodiscard]]`?

Reply:
if I don't, it starts warning on the unit tests.

Reply:
I'd prefer to fix that in the unit tests. Could you suppress the warning with the [cast-to-void](https://stackoverflow.com/a/53581769) trick?

Reply:
eh I don't see the value. All such warnings are in ASSERT_THROW

Reply:
OK. Fixed the unit tests.